flask_monitoringdashboard/database/__init__.py

```diff
@@ -1,264 +1,40 @@
 """Creates the database.
-For information about how to access the database via a session-variable, see: session_scope()
+For information about how to access the database via a DatabaseConnectionWrapper
```

Comment:
this comment is now a sentence that does not have a verb?

Reply:
it should be better now